src/main/scala/cromwell/engine/package.scala

```diff
@@ -107,4 +108,22 @@ package object engine {
     type ExecutionStatus = Value
     val NotStarted, Starting, Running, Failed, Done, Aborted, Aborting = Value
   }
+
+  /*
+   * Type and implicit conversion methods for ExecutionIndex
+   */
+  object ExecutionIndex {
```

Comment:
You'll have to explain to me after the demo in person, I don't quite follow how the implicit class suggestion would look